.storybook/webpack.config.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-underscore-dangle */
+
+/* eslint-disable no-param-reassign */
+
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
```suggestion
/* eslint-disable no-underscore-dangle */
/* eslint-disable no-param-reassign */
/* eslint-disable @typescript-eslint/naming-convention */
```

Reply:
+1

Reply:
This won't be possible, prettier automatically moves it